app-libs/stf/src/lib.rs

```diff
@@ -63,8 +63,6 @@ pub enum StfError {
 	Dispatch(String),
 	#[display(fmt = "Not enough funds to perform operation")]
 	MissingFunds,
-	#[display(fmt = "Account does not exist {:?}", _0)]
```

Comment:
There's no `InexistentAccount`. It simply has balance zero.

Reply:
I agree, we don't need to introduce abstractions at the core level that substrate doesn't have